shared/attributes.asciidoc

```diff
@@ -123,13 +123,21 @@ ifdef::elasticsearch-root[]
 endif::elasticsearch-root[]
 
 //////////
-Kibana app names
+Kibana app and UI names
 //////////
 
-:uptime-app: Uptime app
-:logs-app: Logs app
-:metrics-app: Metrics app
-:siem-app: SIEM app
+:apm-app:            APM app
+:uptime-app:         Uptime app
+:logs-app:           Logs app
+:metrics-app:        Metrics app
+:siem-app:           SIEM app
+:security-app:       Elastic Security app
```

Comment:
FYI @benskelker I added this "security-app" attribute based on what I saw in https://github.com/elastic/stack-docs/pull/1317

Reply:
Thanks @lcawl
@jmikell821 @DonNateR  - FYI

Reply:
@DonNateR can you add attributes for `Elastic Security` and `Elastic Agent` please?